frontend/src/libs/components/profile-settings-sidebar/profile-settings-sidebar.tsx

```diff
@@ -0,0 +1,79 @@
+import { Card, Icon } from '#libs/components/components.js';
+import { IconColor } from '#libs/enums/enums.js';
+import { useAppSelector, useCallback, useState } from '#libs/hooks/hooks.js';
+import { type IconName, type SettingsOption } from '#libs/types/types.js';
+import { type UserAuthResponseDto } from '#packages/users/users.js';
+
+import styles from './styles.module.scss';
+
+const settingsOptions: SettingsOption[] = [
+  {
+    name: 'notification',
+    title: 'Very long text to see if it will break',
+    statusIcon: 'arrow',
+  },
+  { name: 'subscription', title: 'Test', statusIcon: 'arrow' },
+];
+
+const ProfileSettingsSidebar: React.FC = () => {
+  const { fullName } = useAppSelector(({ auth }) => {
+    return {
+      fullName: (auth.authenticatedUser as UserAuthResponseDto).fullName,
+    };
+  });
+
+  const [activeButton, setActiveButton] = useState<IconName | null>(null);
+
+  const handleClick = useCallback((name: IconName) => {
+    return () => {
+      setActiveButton(name);
+    };
+  }, []);
```

Comment:
let's remove this nested functions and use them like () => handleClick(name)
```suggestion
  const handleClick = useCallback((name: IconName) => {
    return setActiveButton(name);
  }, []);
```
```

Reply:
Initially I started with this Idea but linter was quite strict about it, so the nested function was the only solution I could figure out. 

<img width="733" alt="Screenshot 2023-09-10 at 18 13 59" src="https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/62558753/201483e3-5258-495f-8523-75f4e4777816">

Reply:
ok, got it